### PR TITLE
Added footer, modified home image

### PR DESCRIPTION
**Pull request by Bryan-Tam**
Modified the home image so that it could be displayed on home page. Footer is uniformly placed on the bottom of the entire App.js

Preview: https://dev-footer.serratus.io

### CODE REVIEW

File: src/pages/Home.jsx [hunk 1]
```diff
@@ -12,8 +12,10 @@ const Home = () => {
     return (
         <div>
             {headTags}
-            <img src="/serratus.jpg" alt="serratus mountain" className="object-cover h-screen w-screen fixed z-0" />
-            <div className="h-screen w-screen fixed flex flex-col items-center justify-center">
+            <div style={{position:"relative", textAlign:"center"}}>
```
Comment (victorlin) [inline src/pages/Home.jsx:15]: use tailwind classes here

File: src/pages/Home.jsx [hunk 2]
```diff
@@ -12,8 +12,10 @@ const Home = () => {
...
+                <img src="/serratus.jpg" alt="serratus mountain" className="" />
+            </div>
+            <div style={{position:"absolute",top:"50%", left:"50%", transform:"translate(-50%,-50%)"}} className="">
```
Comment (victorlin) [inline src/pages/Home.jsx:18]: tailwind classes

File: src/components/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const Footer = () => {
+    return(
+        <footer>
```
Comment (victorlin) [inline src/components/Footer.jsx:5]: add styles (suggestion: `bg-gray-400 px-4 py-2 border-t-2 border-gray-300`)

File: src/components/Footer.jsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+            This website is still under <a style={{color:"blue"}} href="https://github.com/serratus-bio/serratus.io">active development</a>.
```
Comment (victorlin) [inline src/components/Footer.jsx:6]: use `<ExternalLink>` with `text-blue-600` ([example](https://github.com/serratus-bio/serratus.io/blob/677f0dbc/src/pages/Team.jsx#L25))

**Comment by ababaian** [comment]
It's a little heavy with the soild background / black text. Can we use an ~50% alpha white box with center-aligned text?

**Comment by victorlin** [comment]
> It's a little heavy with the soild background / black text. Can we use an ~50% alpha white box with center-aligned text?

Our new contributor was unfamiliar with the styling method we use, I take blame for not having that documentation in place 😬 Suggested some changes above and will add those docs for the future.

> Also it's probably best to direct everyone to github and not the email. This will prevent you from having to check multiple places and/or getting the exact same request 20 times by email if an issue is open/known already.

Good point - @Bryan-Tam please remove the part about reaching out via email. Thanks!

**Comment by ababaian** [comment]
Awesome! Thanks @Bryan-Tam! Stoked to see you onboard!

**Review by victorlin — APPROVED** [review]

**Pull request merged**